app/controllers/users/registrations_controller.rb

```diff
@@ -52,40 +40,16 @@ def update
      super
    end
 
-  # DELETE /resource
-  # def destroy
-  #   super
-  # end
-
-  # GET /resource/cancel
-  # Forces the session data which is usually expired after sign
-  # in to be expired now. This is useful if the user wants to
-  # cancel oauth signing in/up in the middle of the process,
-  # removing all OAuth session data.
-  # def cancel
-  #   super
-  # end
-
-  # protected
+  protected
 
   # If you have extra params to permit, append them to the sanitizer.
   def configure_sign_up_params
     devise_parameter_sanitizer.permit(:sign_up) do |user|
-    user.permit(:id, :name, :email, :password, :password_confirmation)
+    user.permit(:id, :name, :email, :password, :password_confirmation, :admin)
```

Comment:
What does this do?  Will it allow users to sign themselves up as admins?

Reply:
No! The controller only allows admins to create new account. The ```configure_sign_up_params``` specifies which attributes are allowed to be changed.